app/lib/constituency_api.rb

```diff
@@ -6,9 +6,25 @@ class Constituency < Struct.new(:name, :mp)
 
   class Mp < Struct.new(:id, :name, :start_date)
     URL = "http://www.parliament.uk/biographies/commons"
+
+    def initialize(id, name, start_date)
+      self.id = id.to_i
+      self.name = name
+      self.start_date = parse_date(start_date)
```

Comment:
I haven't seen any cases where it was nil or a non-standard date string, but Jon suggested to try catching any unexpected errors due to it being an external dataset. 
Would you suggest just to leave it for now then and assume that .to_date() always works?